eng/targets/ResolveReferences.targets

```diff
@@ -204,7 +204,7 @@
     <Warning Condition="@(UnusedBaselinePackageReference->Count()) != 0"
              Text="Package references changed since the last release. This could be a breaking change. References removed:%0A - @(UnusedBaselinePackageReference, '%0A -')" />
 
-    <Error Condition="'$(TargetFrameworkIdentifier)' != '.NETFramework' AND '%(Reference.Identity)' != '' AND ! Exists('%(Reference.Identity)')"
+    <Error Condition="'$(TargetFrameworkIdentifier)' != '.NETFramework' AND '%(Reference.Identity)' != '' AND ! Exists('%(Reference.Identity)') AND '$(DisablePackageReferenceRestrictions)' != 'true'"
```

Comment:
What does this do?

Reply:
It disables the restrictions set in this file, which would keep a project from doing things like having a `PackageReference`.